datarepo-clienttests/src/main/java/utils/BulkLoadUtils.java

```diff
@@ -9,33 +12,24 @@
 import java.util.Map;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import utils.DataRepoUtils;
-import utils.FileUtils;
-import utils.KubernetesClientUtils;
-
-public class KubeConfig extends runner.TestScript {
-  private static final Logger logger = LoggerFactory.getLogger(KubeConfig.class);
 
-  /** Public constructor so that this class can be instantiated via reflection. */
-  public KubeConfig() {
-    super();
-  }
+public class BulkLoadUtils {
+  private static final Logger logger = LoggerFactory.getLogger(BulkLoadUtils.class);
 
-  private int filesToLoad;
   private String datasetCreator;
   private BillingProfileModel billingProfileModel;
   private DatasetSummaryModel datasetSummaryModel;
+  private String loadTag;
 
-  public void setParameters(List<String> parameters) throws Exception {
-    if (parameters == null || parameters.size() == 0) {
-      throw new IllegalArgumentException(
-          "Must provide a number of files to load in the parameters list");
-    } else {
-      filesToLoad = Integer.parseInt(parameters.get(0));
+  public String getDatasetId() {
+    if (datasetSummaryModel == null) {
+      throw new UnsupportedOperationException("datasetSummaryModel has not been defined.");
     }
+    return datasetSummaryModel.getId();
   }
 
-  public void setup(Map<String, ApiClient> apiClients) throws Exception {
+  public void bulkLoadSetup(Map<String, ApiClient> apiClients, String billingAccount)
```

Comment:
The setup/cleanup methods in this class feel out of place to me. They're not really bulk load related, in the same way that buildBulkLoadFileRequest and getAndDisplayResults are. I think the profile/dataset creation/deletion should be in the TestScript class instead.